2023-12-29T00:20:02.3245155Z ##[group]Run actions/setup-python@v4
2023-12-29T00:20:02.3245741Z with:
2023-12-29T00:20:02.3246025Z   python-version: 3.8
2023-12-29T00:20:02.3246397Z   architecture: x64
2023-12-29T00:20:02.3246832Z   cache: pip
2023-12-29T00:20:02.3247155Z   check-latest: false
2023-12-29T00:20:02.3247621Z   token: ***
2023-12-29T00:20:02.3248057Z   update-environment: true
2023-12-29T00:20:02.3248449Z   allow-prereleases: false
2023-12-29T00:20:02.3249082Z ##[endgroup]
2023-12-29T00:20:02.4588931Z ##[group]Installed versions
2023-12-29T00:20:02.4662597Z Successfully set up CPython (3.8.18)
2023-12-29T00:20:02.4663798Z ##[endgroup]
2023-12-29T00:20:03.3613850Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-12-29T00:20:03.7184330Z /home/runner/.cache/pip
2023-12-29T00:20:05.3740309Z Received 4194304 of 6245959 (67.2%), 4.0 MBs/sec
2023-12-29T00:20:05.4250407Z Received 6245959 of 6245959 (100.0%), 5.7 MBs/sec
2023-12-29T00:20:05.4251675Z Cache Size: ~6 MB (6245959 B)
2023-12-29T00:20:05.4280243Z [command]/usr/bin/tar -xf /home/runner/work/_temp/e1ab1306-7256-43dc-8ee6-a5de0585606c/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-12-29T00:20:05.4541003Z Cache restored successfully
2023-12-29T00:20:05.4561704Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
